addons/rating/models/rating.py

```diff
@@ -53,12 +70,43 @@ def _compute_rating_image(self):
             except (IOError, OSError):
                 rating.rating_image = False
 
-    @api.multi
     @api.depends('rating')
     def _compute_rating_text(self):
-        text = {10: _('Satisfied'), 5: _('Not satisfied'), 1: _('Highly dissatisfied')}
         for rating in self:
-            rating.rating_text = text[rating.rating] or _('No rating yet')
+            if rating.rating >= 7:
+                rating.rating_text = 'satisfied'
+            elif rating.rating > 3:
+                rating.rating_text = 'not_satisfied'
+            elif rating.rating >= 1:
+                rating.rating_text = 'highly_dissatisfied'
+            else:
+                rating.rating_text = 'no_rating'
+
+    @api.model
+    def create(self, values):
+        if values.get('res_model_id') and values.get('res_id'):
+            values.update(self._find_parent_data(values))
+        return super(Rating, self).create(values)
+
+    @api.multi
+    def write(self, values):
+        if values.get('res_model_id') and values.get('res_id'):
+            values.update(self._find_parent_data(values))
+        return super(Rating, self).write(values)
+
+    def _find_parent_data(self, values):
```

Comment:
We wanted to apply the same mecanism as mail.alias. Do you think it could impact performance ? 
Computed field also means that we would have to override the computed method to find parent. Overriding `rating.rating` in each module can be strange, since we have the mixin.